src/app/pcasts/models/user.py

```diff
@@ -45,5 +45,11 @@ def __eq__(self, other_user):
   @validates('username')
   def validate_username(self, key, username):
     if len(username) < 1:
-      raise Exception("Username length must greater than 0")
+      raise Exception('Username length must greater than 0')
     return username
+
+  @validates('google_id', 'facebook_id')
+  def validate_ids(self, key, value):
+    if (self.google_id, self.facebook_id, value) == (None, None, None):
+      raise Exception('Id\'s can\'t be none')
+    return value
```

Comment:
This should actually be removed, I think this will be called when `self.google_id` gets a value assigned to it, and at that point `facebook_id` is still None, so if the account is being created w/ facebook, it will throw the error. That's why the assert statement is at the top of the __init__ as a temporary workaround

Reply:
Ok I'll remove this entire function - it's a relic of me merging in Daniel's changes prematurely.